interpolate_phase.py

```diff
@@ -0,0 +1,242 @@
+    ps_msk = (coh > coh_thresh).astype('bool')
+
+    # Get the coordinates of the pixels to be interpolated and generate
+    # a circular sample to interpolate from.
+    centers = np.where(ps_msk == 0)
+    cents_num = len(centers[0])
+    print('A total of ', cents_num, 'pixels require interpolation')
+
+    # Plot PS pixels overview
+    print('Plotting PS pixels overview...')
+    fig = plt.figure(figsize=(7, 5))  # New figure
+    ax = fig.add_subplot(1, 2, 1)  # subplot
+    cax = ax.imshow(ps_msk, cmap='jet', interpolation='nearest')
+    ax.set_title(r'ps_mask')
+    cbar = fig.colorbar(cax, orientation='horizontal')
+    ax = fig.add_subplot(1, 2, 2)  # subplot
+    cax = ax.imshow(np.ma.masked_where(np.angle(w_int) * ps_msk == 0,
+                                       np.angle(w_int) * ps_msk),
+                    cmap='hsv', interpolation='nearest')
+    ax.set_title(r'ps_pixels')
+    cbar = fig.colorbar(cax, orientation='horizontal')
+    plt.savefig('ps_pixels_overview.png', dpi=300, facecolor='white')
+    plt.close()
+
+    return ps_msk
+
+
+def run_chen_interp(w_int, ps_msk):
+    """Run interpolation for test array."""
+    from dev import phase_interp as phint
+
+    t_all = time.time()  # track processing time
+
+    # Run interpolation here !!!
+    # Modify radius to increase resampling valid region.
+    ifg_re = np.copy(np.angle(w_int) * ps_msk)  # array to be resampled
+    # run interpolation
+    ifg_re = phint.chen_interp(ifg_re, radius=20, fill_value=0.0)
+
+    # Track processing time
+    t_all_elapsed = time.time() - t_all  # track processing time
+    hours, rem = divmod(t_all_elapsed, 3600)
+    minutes, seconds = divmod(rem, 60)
+    print(f"Successfully interpolated phase in {hours:0>2}:{minutes:0>2}:"
+          f"{seconds:05.2f} hours:min:secs")  # track processing time
+
+    return ifg_re
+
+
+def run_chen_interp_v2(w_int, ps_msk):
+    """Run interpolation for test array."""
+    from dev import phase_interp as phint
+
+    t_all = time.time()  # track processing time
+
+    # Run interpolation here !!!
+    # Modify radius to increase resampling valid region.
+    ifg_re = np.copy(np.angle(w_int) * ps_msk)  # array to be resampled
+    # run interpolation
+    ifg_re = phint.chen_interp_v2(ifg_re, radius=20, fill_value=0.0)
+
+    # Track processing time
+    t_all_elapsed = time.time() - t_all  # track processing time
+    hours, rem = divmod(t_all_elapsed, 3600)
+    minutes, seconds = divmod(rem, 60)
+    print(f"Successfully interpolated phase in {hours:0>2}:{minutes:0>2}:"
+          f"{seconds:05.2f} hours:min:secs")  # track processing time
+
+    return ifg_re
+
+
+def run_interp_numba(w_int, ps_msk):
+    """Run interpolation for test array."""
+    print('Running interpolation with "interp_numba"')
+    from dev import interp_numba
+
+    t_all = time.time()  # track processing time
+
+    # run interpolation
+    ifg_re = interp_numba.interp(w_int, ps_msk,
```

Comment:
I think we should expose the options here, because maybe a user wants to increase radius or number of valid pixels etc.